packages/manager/modules/anthos/src/dashboard/components/assign-private-ip/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Every mention of `IP` should be preceed by `Adresse(s)`

Reply:
I used privateIp to be compliant with API side.
The use privateIp as model name.